gozw.go

```diff
@@ -325,6 +323,33 @@ func (c *Client) AddNode() (*Node, error) {
 		}
 	}
 
+	progress <- PairingProgressUpdate{
+		InterviewedCommandClassCount: 0,
+		ReportedCommandClassCount:    len(node.CommandClasses),
+	}
+
+	c.l.Debug("reported command classes", zap.Int("len", len(newNodeInfo.CommandClasses)))
+
+
+	go func() {
```

Comment:
Might be good to pass some kind of ctx in here so that we don't leak goroutines for failed pairing sessions.